shared/js/background/utils.es6.js

```diff
@@ -7,9 +7,16 @@ const browserInfo = parseUserAgentString()
 function extractHostFromURL (url, shouldKeepWWW) {
     if (!url) return ''
 
-    let urlObj = tldjs.parse(url)
+    const urlObj = tldjs.parse(url)
```

Comment:
`tldjs` doesn't support IPv6, so this function was always returning an empty string before. My fix is… OKish.